gke/README.md

```diff
@@ -51,8 +51,36 @@ The following describes the setup process for this project:
 1. Install Jenkins on GKE using the stable/jenkins helm chart: [Jenkins On GKE Tutorial](
    https://cloud.google.com/solutions/jenkins-on-kubernetes-engine-tutorial).
 
-1. Install and configure the GKE plugin: [Jenkins GKE Docs](
-   https://github.com/jenkinsci/google-kubernetes-engine-plugin/blob/develop/docs/Home.md).
+1. Install and configure the following plugins:
+   1. GKE plugin: [Jenkins GKE Docs](https://github.com/jenkinsci/google-kubernetes-engine-plugin/blob/develop/docs/Home.md).
+   1. GCS plugin: [Jenkins GCS Docs](https://github.com/jenkinsci/google-storage-plugin/blob/develop/README.md).
+   1. Global Variable String Parameter plugin: [Jenkins Global Var Docs](https://wiki.jenkins.io/display/JENKINS/Global+Variable+String+Parameter+Plugin).
+   1. Blue Ocean: [Blue Ocean Docs](https://jenkins.io/doc/book/blueocean/getting-started/#on-an-existing-jenkins-instance).
```

Comment:
Similar to above, the Blue Ocean plugin isn't required for this example. If customers want to use it that's fine, but I don't want to make it seem like it's required for this project to function properly.

Reply:
That's fine, and I'd love us to simplify actually.  Previously the README linked to https://jenkins.io/doc/tutorials/build-a-multibranch-pipeline-project/ which requires it.

Reply:
I didn't realize they did, I can see how that would be misleading. But confirmed, Blue Ocean is not required.

Reply:
Okay, let uninstall and rewrite these steps a little bit.